Dockerfile

```diff
@@ -1,4 +1,9 @@
-FROM python:2.7
+FROM ubuntu:16.04
+RUN apt update && apt -y install usbutils software-properties-common python-pip
+RUN echo -e "\n" | add-apt-repository ppa:rolfbensch/sane-release
+RUN apt-get update
+RUN apt -y install libsane=1.0.27-xenial1 sane-utils=1.0.27-xenial1 libsane-common=1.0.27-xenial1
+RUN echo "usb 0x4b8 0x151" > /etc/sane.d/epson2.conf
```

Comment:
Use `>>` rather than `>` and please test to add the other scanner's codes as well and see if it works.